src/rax.h

```diff
@@ -206,6 +207,7 @@ void raxStop(raxIterator *it);
 int raxEOF(raxIterator *it);
 void raxShow(rax *rax);
 uint64_t raxSize(rax *rax);
+size_t raxAllocSize(rax *rax);
```

Comment:
we should certainly use that in the MEMORY USAGE commands, that's the whole purpose of this work.. right?